test/WebJobs.Script.Tests/Extensions/EnvironmentExtensionsTests.cs

```diff
@@ -256,21 +263,32 @@ public void IsAnyKubernetesEnvironment_ReturnsExpectedResult(bool isKubernetesMa
                 env.SetEnvironmentVariable(ManagedEnvironment, "true");
             }
 
+            if (isConnectedAppEnvironment)
+            {
+                env.SetEnvironmentVariable(ConnectedEnvironment, "true");
+            }
+
             Assert.Equal(expectedValue, env.IsAnyKubernetesEnvironment());
         }
 
         [Theory]
-        [InlineData(true, true)]
-        [InlineData(false, false)]
-        public void IsManagedAppEnvironment_ReturnsExpectedResult(bool isManagedAppEnvironment, bool expectedValue)
+        [InlineData(true, false, true)]
+        [InlineData(false, true, true)]
+        [InlineData(false, false, false)]
```

Comment:
Looks like you're missing a case - both can be true, and per the implementation it should return true

Reply:
Will update this case.

Reply:
Thanks. While I see from your other explanation that in practice both won't be set, it's still good to have the test match the code